apstools/devices/hhl_slits.py

```diff
@@ -0,0 +1,68 @@
+from ophyd import Component as Cpt
+from ophyd import Device
+from ophyd import FormattedComponent as FCpt
+from ophyd import EpicsMotor
+
+
+class HHLSlits(Device):
+    """
+    High Heat Load Slit OPHYD Device.
```

Comment:
Could drop the text `OPHYD Device`.  That could apply to just about any code in this directory.
![image](https://github.com/BCDA-APS/apstools/assets/2279984/5ebff2ea-dcf1-440a-8a36-145bd231c051)

Reply:
This proposition is your call.  If you wish, change it before merging.